_data/simple-icons.json

```diff
@@ -13295,20 +13300,20 @@
 		"aliases": {
 			"dup": [
 				{
-					"title": "France Inter",
```

Comment:
These changes appear to be coming from the commit hook, I've tried to undo them but perhaps this is correct?